### PR TITLE
[1] First commit.

### PR DESCRIPTION
**Pull request by alainmenag**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

**Comment by alainmenag** [comment]
I've committed 2 new changes. Do I need to close this pull request and create a new one?

File: connectors/Telephony-Xtended-Service/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2445 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Telephony Xtended Serv Interf",
+    "description": "Integrate with BluIP's telephony system to take your call to the next level. Telephony Xtended Service Interface (XSI).",
+    "version": "1.0"
```
Comment (v-sriyen) [inline connectors/Telephony-Xtended-Service/apiDefinition.swagger.json]: @alainmenag Please add the x-ms-api-annotation in the info, Refer the swagger we have shared with you 

"x-ms-api-annotation": {
      "status": "Preview"
    }

File: connectors/Telephony-Xtended-Service/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2445 @@
+{
```
Comment (v-sriyen) [inline connectors/Telephony-Xtended-Service/apiDefinition.swagger.json]: @alainmenag Please delete below unwanted files from the commit

.DS_Store
connectors/.DS_Store
connectors/Telephony-Xtended-Service/.DS_Store

File: connectors/Telephony-Xtended-Service/apiDefinition.swagger.json (hunk 1 above)
Comment (alainmenag) [reply connectors/Telephony-Xtended-Service/apiDefinition.swagger.json]: All done. I had a confusion of files. Thank you.

File: connectors/Telephony-Xtended-Service/apiDefinition.swagger.json (hunk 2 above)
Comment (alainmenag) [reply connectors/Telephony-Xtended-Service/apiDefinition.swagger.json]: Removed.

**Pull request merged**